### ISSUE TITLE
how to display a simple selection zim file as a list not with images . .

### ISSUE THREAD

**Issue by hamoudak** [openzim/mwoffliner#2087]
I've made a zim file with wp1.openzim.org about "herbal medicine", but it displays articles in the interface with images. so I wanna to be listed as a list with no images, because it's a long one and it seems not practical for me ; how to do that?
<img width="956" alt="interface-with-images" src="https://github.com/user-attachments/assets/4d1d3eed-0831-478e-b993-f02b240707d7">

<img width="956" alt="interface-with-images2" src="https://github.com/user-attachments/assets/7bfab7c5-2986-4212-9f2b-c3fdf1a3732d">

**Comment by Popolechien** [comment]
@kelson I don't think this answered OP's question, which if I am correct would rather be: how can we see the list of entries in this zim file as text, rather than a (limited) selection of clickable images?

**Comment by hamoudak** [comment]
yes, both answers are correct. lets say I want to make a project about diseases, it won't be appropriate or uncomfortabe to my eyes and distracting at the same time. I do not want it to be limited to clickable images. before I had bullet point list zims and it was good . could this be a feature in app settings?

**Comment by kelson42** [comment]
@hamoudak I you have a very limited number of articles, you will get a bullet list; but this is actually a bug/limitation I would like to fix. What you want it getting a bullet list *in place* of the welcome page with thumbs. I'm not against this feature request but (1) I don't really know how to implement this elegantly (ie. without adding an option) (2) this is definitly not part of my priorities of the moment.

**Comment by hamoudak** [comment]
I see for the first line you said; if you have a very limited number of articles you get images is bettert, then you have bullet list; what do you think?

**Comment by kelson42** [comment]
I come back on this issue and rereading it I think I will close it as won't fix.

What is requested here is a full (I guess) sorted list of all articles titles (with links to the article content), probably with sections based on starting letters. Kind of like in a paper dictionary.

I don't think this is a good idea because it is basically the suggestions allows to go through that list in a far better manner than a real list. This issue also does not explain what is the problem with the current solution (beside " distracting at the same time")

**Comment by hamoudak** [comment]
there's no problem with the images as the default for the app, I just see unprober images  like"naked stuff" when I make for example medical content, for now I have rearranged my titles list, so it won't show first at the landing page "gallery". I meant first like"ray charles" zim file but with limited bullets as the images are.

**Comment by kelson42** [comment]
@hamoudak The tiles are the one for 100 (or so) first articles of the list, so you can perfectly decide what kind of tiles you get.

**Comment by hamoudak** [comment]
I have tried this with myself. didn't know "the number" exactly. I guessed about 70 titles, thanks for telling me this.

**Comment by benoit74** [comment]
Having illustrations on landing page also means that loading the ZIM induces a significant memory / bandwidth consumption. 

One might prefer to simply have a list of the top 100 articles, just like we currently display the top 100 images. I don't see where this is a big problem to implement.

And I second the fact that having to manually sort the articles list to ensure we do not have inappropriate / disturbing / hurting images is a pain. All wikipedia_*_medicine ZIMs would be very disgusting should we not have implementing a custom landing page.

Btw, this is another alternative when you have write access to the wiki: create your own landing page in the wiki. This gives you full control on the look and feel of the ZIM landing page. But of course, this is not always possible.
